BitwardenShared/UI/Vault/Vault/VaultList/VaultListView.swift

```diff
@@ -345,6 +346,20 @@ struct VaultListView: View {
         .task(id: store.state.vaultFilterType) {
             await store.perform(.streamVaultList)
         }
+        .task(id: store.state.isEligibleForAppReview) {
+            if store.state.isEligibleForAppReview {
+                SKStoreReviewController.requestReview()
```

Comment:
we can replace this with when we bump the version to iOS 16.
```
    @Environment(\.requestReview) var requestReview
    requestReview()
```

Reply:
🤔 Is there a way we could use a `#avaialble` check here to use the controller vs the environment approach one? Because `SKStoreReviewController` has been [deprecated in iOS 18](https://developer.apple.com/documentation/storekit/skstorereviewcontroller).

Reply:
this has been updated.